lib/tokens.js

```diff
@@ -449,6 +469,64 @@ export async function resolveToInternalId({token}) {
   return {internalId};
 }
 
+/**
+ * Sets the `minAssuranceForResolution` on the token batches associated
+ *   `internalId`.
```

Comment:
We don't indent function descriptions.

```suggestion
 * `internalId`.
```